webapp/src/ts/modules/contacts/contacts-filters.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="basic-filters left-pane">
+<div class="basic-filters left-pane only-filter">
```

Comment:
I think it would make more sense to add this class to the `mm-freetext-filter` component underneath. 

The css rule would also need to change to: 
```
.filter.only-filter, .only-filter .freetext-filter, {
    display: inline-block;
    width: auto;
}
```

What do you think?

Reply:
Hi, as I understand, this class affects the display and it's intended for when there's only 1 filter (regardless of the type) in the page, right?
The first option is to leave the responsibility of determining the filter display to the container, something like every page defines the layout as needed and keep the little components basic. 
The other option is to add the class to every filter component, the class will be "turned on/off" by an input value sent from the container. By sending the input value, we minimise specific logic (like contacts) in the future and keep the reusable components simple. 
I think I prefer the first one.

Reply:
Then the CSS class name is confusing. When applied to a "filter", it would signify that this is the "only filter" that is displayed. But when applied to the container it's just confusing: what can it mean in this case?

Reply:
oh... 🤔 container has one filter at the time so: one-filter, has-one-filter 😶

Reply:
I've added `show-one-filter` class to the container.